CONTRIBUTING.md

```diff
@@ -1,53 +1,102 @@
-# Contributing Guide
+# DSNP Example Client: Contribution Guide
 
-## Welcome
-Welcome to the example client for demonstrating the use of the DSNP (Distributed Social Networking Protocol) SDK. Thank you for opening this file and considering contributing to the project either by reporting issues or directly contributing code to the project! First, check out our project wide [Contributing Standards](https://github.com/LibertyDSNP/liberty-web/blob/main/CONTRIBUTING.md) and then read on below for specifics about this project.
+Thank you for considering contributing, whether by reporting issues,
+improving documentation, or contributing code.
+
+First, please see the [Contribution
+Guide](https://github.com/LibertyDSNP/liberty-web/blob/main/CONTRIBUTING.md)
+for Project Liberty as a whole.  Below are specifics about the Example
+Client project.
 
 ## References
-In creating bug reports, feature requests, or pull requests it's important to be familiar and up to date with both: 
+
+DSNP is a fast-moving environment.  When creating bug reports, feature
+requests, or pull requests, it's important to be familiar and
+up-to-date with:
+
 * [DSNP Specs](https://github.com/LibertyDSNP/spec) for how the DSNP works
+
 * [DSNP SDK](https://github.com/LibertyDSNP/sdk-ts) for integration with the DSNP
 
 ## Reporting
 
 ### Bugs
-If you are using the example client and you find a bug that you believe should be fixed, please create a new issue on our [Github Issues Page](https://github.com/LibertyDSNP/example-client/issues). Make sure to **tag it as a bug**, and follow the `Bug Report Template`. For all intents and purposes a Bug is any one of the following things:
-* Broken/Nonfunctional code
-* Unexpected/Unintentional results
-* Code vulnerabilities
-* Out of date code/packages/documentation
-* Documentation/Code inconsistencies
+
+If you are using the Example Client and you find a bug that you
+believe should be fixed, please [create a new issue
+ticket](https://github.com/LibertyDSNP/example-client/issues/new/choose)
+(after first [looking to see if there is an existing
+issue](https://github.com/LibertyDSNP/example-client/issues) about it,
+of course).  Make sure to **tag the new issue as a bug**, and follow
+the `Bug Report` template.
+
+A "bug" is any one of the following things:
```

Comment:
I don't think we need to define what a bug is.  Just about anyone coming to the GitHub repo will not need this information.

Reply:
Yeah, good point.